webapp/channels/src/components/admin_console/system_user_detail/system_user_detail.tsx

```diff
@@ -374,21 +512,109 @@ export class SystemUserDetail extends PureComponent<Props, State> {
                                         <EmailIcon/>
-                                        <input
-                                            className='form-control'
-                                            type='text'
-                                            value={this.state.emailField}
-                                            onChange={this.handleEmailChange}
-                                            disabled={this.state.error !== null || this.state.isSaving}
-                                        />
+                                        {this.state.user?.auth_service ? (
+                                            <WithTooltip
+                                                title={this.props.intl.formatMessage({
+                                                    id: 'admin.userDetail.managedByProvider.title',
+                                                    defaultMessage: 'Managed by login provider',
+                                                })}
+                                                hint={this.props.intl.formatMessage({
+                                                    id: 'admin.userDetail.managedByProvider.email',
+                                                    defaultMessage: 'This email is managed by the {authService} login provider and cannot be changed here.',
+                                                }, {
+                                                    authService: this.state.user.auth_service.toUpperCase(),
+                                                })}
+                                            >
+                                                <input
+                                                    className='form-control'
+                                                    type='text'
+                                                    value={this.state.emailField}
+                                                    disabled={true}
+                                                    readOnly={true}
+                                                    style={{cursor: 'not-allowed'}}
+                                                />
+                                            </WithTooltip>
+                                        ) : (
+                                            <>
+                                                <input
+                                                    className={classNames('form-control', {
+                                                        error: this.state.emailError,
+                                                    })}
+                                                    type='text'
+                                                    value={this.state.emailField}
+                                                    onChange={this.handleEmailChange}
+                                                    disabled={this.state.error !== null || this.state.isSaving}
```

Comment:
The first case of this looks like it disables the textbox whenever there's an error which seems wrong to me even if that's how it worked before